service/modifier.go

```diff
@@ -0,0 +1,59 @@
+package service
+
+import (
+	"net/http"
+)
+
+// Edits the response before proxying it back.
+// If Auth tokens are present then set them as cookies
+func identityResponseModifier(r *http.Response) error {
+	// maxAgeBrowserSession is length of time to expire cookie on browser close
+	maxAgeBrowserSession := 0
+	if r.StatusCode >= 200 && r.StatusCode < 300 {
```

Comment:
Since your're importing the `net/http` package, its probably worth using the the `http` constants rather than inlining the codes:

```
if r.StatusCode >= http. StatusOK && r.StatusCode < http. StatusMultipleChoices {
```

Reply:
Good shout